packages/ui/components/ActiveLink.tsx

```diff
@@ -17,12 +17,12 @@ const ActiveLink = ({ children, activeClassName, ...props }) => {
       : `${props.className} ${childClassName}`
 
   return (
-    <Link href={props.href} {...props}>
+    <Link href={props.href} {...props} legacyBehavior>
```

Comment:
what's this legacy behavior thing?

Reply:
Has to do with NextJS Links no longer requiring an <a> within them?
https://nextjs.org/docs/pages/api-reference/components/link#legacybehavior

Reply:
a problem for later.